code/datums/tips/tips.dm

```diff
@@ -182,7 +239,11 @@ GLOBAL_LIST_EMPTY(tips_by_category)
 		"As a Security Officer, communicate and coordinate with your fellow officers using the security channel (:s) to avoid confusion.",
 		"As a Security Officer, your sechuds or HUDsunglasses can not only see crewmates' job assignments and criminal status, but also if they have a mindshield implant. Use this to your advantage in a revolution to definitively tell who is on your side!",
 		"As a Security Officer, examining someone while wearing sechuds or HUDsunglasses will let you set their arrest level.",
-		"As the Investigator, you can dust almost any surface for prints. People leave fingerprints on everywhere they touch unless they are wearing gloves. However, as gloves leave fibres specific to their type, such as black or nitrile, it is possible to guess what general department they come from."
+		"As a Security Officer, you're not here to kill your coworkers. Try non lethal ammo and weapons first.",
+		"As a Security Officer, you can stun people without doing brute damage, by attacking with the help intent. You can do both stun and do brute damage by attacking on harm intent.",
```

Comment:
This needs to say it's for the stunbatoon, otherwise it seems you can do that with only your hands

Reply:
Changed this line to be a bit more clear and flow better